api/src/test/java/io/opentelemetry/metrics/DefaultMeterTest.java

```diff
@@ -56,6 +56,11 @@ public void noopAddMeasureLong_NullName() {
     defaultMeter.longMeasureBuilder(null);
   }
 
+  @Test
+  public void noopAddMeasureLong_EmptyName() {
```

Comment:
Add an issue about this, I don't think we should accept empty name, but we can address this separately.

Reply:
I was about to create the issue :wink: